test/integration/features/config.feature

```diff
@@ -9,67 +9,99 @@ Checks whether CRC `config set` command works as expected in conjunction with `c
+        And unsetting config property "<property>" succeeds
         
         # always return to default values
         @darwin
         Examples: Config settings on Mac 
-            | property  |   value1 |  value2 |
-            | cpus      |        4 |       3 |
-            | memory    |     8192 |    4096 |
-            | vm-driver | hyperkit | libvirt |
+            | property         |    value1 |            value2 |
+            | cpus             |         4 |                 3 |
+            | memory           |      8192 |              4096 |
+            | vm-driver        |  hyperkit |           libvirt |
+            | nameserver       | 120.0.0.1 |   999.999.999.999 |
+            | pull-secret-file |      /etc | /nonexistent-file |
 
         @linux
         Examples: Config settings on Linux
-            | property  |  value1 |   value2 |
-            | cpus      |       4 |        3 |
-            | memory    |    8192 |     4096 |
-            | vm-driver | libvirt | hyperkit |
+            | property         |    value1 |            value2 |
+            | cpus             |         4 |                 3 |
+            | memory           |      8192 |              4096 |
+            | vm-driver        |   libvirt |          hyperkit |
+            | nameserver       | 120.0.0.1 |   999.999.999.999 |
+            | pull-secret-file |      /etc | /nonexistent-file |
 
         @windows
         Examples: Config settings on Windows
-            | property  | value1                   | value2                                               |
-            | cpus      | 4                        | 3                                                    |
-            | memory    | 8192                     | 4096                                                 |
-            | vm-driver | hyperv                   | hyperkit                                             |
+            | property         |    value1 |            value2 |
+            | cpus             |         4 |                 3 |
+            | memory           |      8192 |              4096 |
+            | vm-driver        |    hyperv |          hyperkit |
+            | nameserver       | 120.0.0.1 |   999.999.999.999 |
+            | pull-secret-file |    /Users | /nonexistent-file |
 
     @linux @darwin @windows
     Scenario: CRC config checks (bundle version)
         When setting config property "bundle" to value "current bundle" succeeds
         And "JSON" config file "crc.json" in CRC home folder contains key "bundle" with value matching "current bundle"
         And setting config property "bundle" to value "/path/to/nonexistent/bundle/crc_hypervisor_version.tar.xz" fails
 
+    @linux @darwin @windows
+    Scenario: CRC config checks (update check)
+        Given setting config property "disable-update-check" to value "true" succeeds
+        When executing "crc setup" succeeds
+        Then stdout should not contain "new version" 
```

Comment:
Shouldn't we never get 'new version' in the output? Integration will most of the time test either a new release, or a git version, in both cases there will not be any newer version available even without setting the configuration key?
I think right now there's a bug where versions built from git output this warning, but if we fix it, then this check will always succeed, so I don't know how good it is to add it?

Reply:
> I think right now there's a bug where versions built from git output this warning
Ah, that's a bug. Good :). I can remove it then, true.

Reply:
I consider this to be a bug, I don't know what others in the team think about it :)

Reply:
it is a bug... git versions should never be checked against the released versions.
this was reported before, but forgot to add an issue... sorry

Reply:
https://github.com/code-ready/crc/issues/806